interface-definitions/include/accel-ppp/ppp-options-ipv6-interface-id.xml.i

```diff
@@ -0,0 +1,48 @@
+<!-- include start from accel-ppp/ppp-options-ipv6-interface-id.xml.i -->
+<leafNode name="ipv6-intf-id">
+  <properties>
+    <help>Fixed or random interface identifier for IPv6</help>
+    <completionHelp>
+      <list>random</list>
+    </completionHelp>
+    <valueHelp>
+      <format>random</format>
+      <description>Random interface identifier for IPv6</description>
+    </valueHelp>
+    <valueHelp>
+      <format>x:x:x:x</format>
+      <description>specify interface identifier for IPv6</description>
+    </valueHelp>
```

Comment:
missing regex constraint probing for `random` and the IPv6 interface qualifier.

Reply:
I got the original XML and just move it to the include :)
I’ll take a look

Reply:
Do I need regex which includes `random` and something like 'ws:wa:wb:ww' ?
At least I can configure it
```
set vpn l2tp remote-access ppp-options ipv6-intf-id 'ws:wa:wb:ww'
```

Reply:
Regex should be: `(random|(([0-9a-fA-F]){1,4}:){3}([0-9a-fA-F]){1,4})`

Reply:
Done!
Due to [accel-pp-manual](https://docs.accel-ppp.org/en/latest/configuration/ppp.html)
```
ipv6-intf-id=x:x:x:x|random

    By default is fixed.

    Specify fixed or random interface identifier for IPv6.
ipv6-peer-intf-id=x:x:x:x|random|ipv4|calling-sid

    By default is fixed.

    Specifies peer interface identifier for IPv6.

    random - generate random interface identifier for peer.

    ipv4 - calculate interface identifier from IPv4 address, for example 192:168:0:1

    calling-sid - calculate interface identifier from Calling-Station-Id.
ipv6-accept-peer-intf-id=0|1
```